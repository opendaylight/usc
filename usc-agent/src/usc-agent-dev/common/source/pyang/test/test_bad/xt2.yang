module xt2 {
  namespace "xt2"; // error: illegal urn
  prefix "xt2";
  organization "hopp";
  contact "hej";
  description "arne" + /* hmm*/ ' ' + 'anka';
  revision "2008-01-01" { description "initinla"; }
}
  

