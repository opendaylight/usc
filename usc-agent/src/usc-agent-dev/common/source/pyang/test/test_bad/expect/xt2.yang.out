xt2.yang:2: error: BAD_VALUE
